### DISCUSSION TITLE
Opening mouse properties using a different method

### DISCUSSION THREAD

**Discussion by fastfingertips** [antiden/macOS-cursors-for-Windows#33] [Ideas]
Mouse properties can also be opened like this: Hold down the Windows Key and Press R on your keyboard. Once you've got the Run command box open, type main.cpl and press Enter. Maybe you want to add this to the readme.md file.

**Comment by antiden** [comment]
Hi @FastFingertips thx for help, but now in dev version, i'm working on automatic installation and replacement of all necessary files. Like this:

<img src="https://antiden.ru/mouse.gif" width="400">

**Comment by antiden** [comment]
@FastFingertips  Ready. Auto install cursors in New pack. You can download it [here](https://github.com/antiden/macOS-cursors-for-Windows/releases/tag/0.4)
